README.md

```diff
@@ -12,7 +12,7 @@
 <!-- Badge section -->
 <div align="center">
     <a href="#">
-        <img src="https://img.shields.io/badge/Python-3.7, 3.8, 3.9, 3.10-efefef">
+        <img src="https://img.shields.io/badge/Python-3.8, 3.9, 3.10-efefef">
```

Comment:
Same as `index.md`, we can instead use https://img.shields.io/badge/Python-3.8+-efefef for more supported Python versions